src/couch_query_servers.erl

```diff
@@ -296,7 +296,8 @@ filter_docs(Req, Db, DDoc, FName, Docs) ->
     #httpd{} = HttpReq ->
         couch_httpd_external:json_req_obj(HttpReq, Db)
     end,
-    JsonDocs = [couch_doc:to_json_obj(Doc, [revs]) || Doc <- Docs],
+    Limit = config:get_integer("couchdb", "filter_revs_limit", 20),
```

Comment:
I went with `couchdb` because `revs_limit` is part of the `db_header` record, but as this is for options going through the query server, that switch to `query_server_config` seems reasonable to me.